raiden/smart_contracts/ChannelManagerLibrary.sol

```diff
@@ -11,6 +11,11 @@ library ChannelManagerLibrary {
         Token token;
     }
 
+    event ChannelDeleted(
```

Comment:
Keep the event only in the contract since this is what is being done in the rest of the codebase